### PR TITLE
[AC-412] Explain register patient process

### PR DESCRIPTION
**Pull request by f4ww4z**
https://issues.openmrs.org/browse/AC-412

Added info on how to register a patient, online and offline.

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -8,11 +8,35 @@
 
 * ### Registering Patients
 
-   1. Entering Patient Details
```
Comment (dkayiwa) [inline features.md:23]: Why did you remove the above header?

File: features.md [hunk 2]
```diff
@@ -8,11 +8,35 @@
...
+   You can register new patients by clicking on the **Register Patient** button in the home menu. When online, created patients are automatically synchronized to the server wih a unique patient ID. When offline, created patients are saved in the user's local database and are synced when the user is connected to the internet. See [Working Offline](#working-offline) for more details.
 
-   2. Adding Patient Photo
```
Comment (dkayiwa) [inline features.md]: Same as above

File: features.md [hunk 3]
```diff
@@ -8,11 +8,35 @@
...
+   #### 1. Entering Patient Details
 
-   3. Finding Similar Patients
```
Comment (dkayiwa) [inline features.md:27]: Same as above

File: features.md (hunk 1 above)
Comment (f4ww4z) [reply features.md:23]: Actually I added header tags to make the section clearer. [Here](https://github.com/openmrs/openmrs-android-client-user-guide/pull/16/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R13)

File: features.md (hunk 2 above)
Comment (f4ww4z) [reply features.md]: Changed to a header [here](https://github.com/openmrs/openmrs-android-client-user-guide/pull/16/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R29)

File: features.md (hunk 3 above)
Comment (f4ww4z) [reply features.md:27]: Changed to a header [here](https://github.com/openmrs/openmrs-android-client-user-guide/pull/16/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R35)

**Comment by dkayiwa** [comment]
@shivtej1505 what do you think of this?

File: features.md [hunk 4]
```diff
@@ -20,11 +20,35 @@
...
+
+   ![Required fields and patient summary](assets/openmrs_android_registering_patients_2.png)
+
+   After successfully adding a new patient, a summary regarding patient's data along with the patient photo is shown. From here you can start to interact with the patient, such as visiting the patient and entering biometrics data in the patient's vitals. These events are easily accessed by either swiping right/left or clicking on the event's tab. Take a look at [Viewing Patients](#viewing-patients) for more things to do with a patient.
+
+   You can also update or delete the patient by heading to the menu and clicking **Update Patient** or **Delete Patient**.
+
+   #### 2. Adding Patient Photo
+
+   ![Patient photo](assets/openmrs_android_registering_patients_3.png)
```
Comment (shivtej1505) [inline features.md:43]: Can we use a different photo here? Glasses don't see good in my personal opinion :)

File: features.md [hunk 5]
```diff
@@ -20,11 +20,35 @@
...
+
+   A cool feature in the Android client during registration is the ability to upload a photo related to the patient. To do this, click on the camera icon. You can either take a photo now or choose an existing photo in your phone/tablet. The photo will then be cropped to match the requried dimensions.
+
+   #### 3. Finding Similar Patients
+
+   ![Similar patients](assets/openmrs_android_registering_patients_4.png)
```
Comment (shivtej1505) [inline features.md:49]: In this image we should actually display a screenshot which shows a list of the similar patients, so trying writing some common name like `Paul` or something. I'm sure `Similar Patient found` will display a list.

**Review by shivtej1505 — COMMENTED** [review]
Sorry, @f4ww4z for the delayed review. Got caught up in multiple things.

File: features.md (hunk 4 above)
Comment (f4ww4z) [reply features.md:43]: What kind of images do you think suit this? A random CC approved photo of a person or an icon of a person?

Comment (shivtej1505) [reply features.md:43]: Any random CC approved photo would do.

**Comment by f4ww4z** [comment]
@shivtej1505 Don't worry about being late. Will work on this soon

**Comment by f4ww4z** [comment]
@shivtej1505 I have made your required changes. I changed the photo to a CC approved photo instead.

**Comment by shivtej1505** [comment]
Great work @f4ww4z, merging!!!
